ntoskrnl/mm/ARM3/section.c

```diff
@@ -3341,7 +3341,7 @@ ULONG
 NTAPI
 MmDoesFileHaveUserWritableReferences(IN PSECTION_OBJECT_POINTERS SectionPointer)
 {
-    UNIMPLEMENTED;
+    static ULONG Warn; if (!Warn++) UNIMPLEMENTED;
```

Comment:
You can just simply use `UNIMPLEMENTED_ONCE` to print such unimplemented debug message thing only once.

Reply:
Indeed, thanks. :smile:

Reply:
Fixed. :ok_hand: